examples/run_pulse_calibration.py

```diff
@@ -42,7 +42,7 @@
 # {{{ add file saving parameters to config dict
 config_dict["type"] = "pulse_calib"
 config_dict["date"] = datetime.now().strftime("%y%m%d")
-config_dict["misc_counter"] += 1
+config_dict["pulse_calib_counter"] += 1
```

Comment:
can be done now that I've generalized the counter variables.

Reply:
save_data does this so I edited accordingly but did change name of counter as you point out

Reply:
deleting this was a mistake on my end and this is needed so that the counter shows up in the active ini after writing. My bad!